### ISSUE TITLE
Move SOLR Config from Bitnami container to address M1 chip issue

### ISSUE THREAD

**Issue by jeremyf** [scientist-softserv/britishlibrary#201]
tl;dr - I think BritishLibrary needs the Bitnami mitigation.

I’m working at getting British Library running locally.  I cloned the github repository and then ran `sc up`; this errored out with the message `container for service "solr" is unhealthy`.  I then ran `docker compose down` followed by `docker compose up`; I wanted more detailed logs.  Of interest is the beginning steps for firing up SOLR:

```
solr 14:02:47.38
solr 14:02:47.40 Welcome to the Bitnami solr container
```

Wondering if I could get some help or guidance on this; I can run diffs against other mitigated projects but figure someone might have an easier time with this.  It looks like the major work is in the `docker-compose.yml` file.

The following commit updated Adventist's docker configuration to address Bitnami mitigation: https://github.com/scientist-softserv/adventist-dl/commit/b53bbc6b3b827423f8b7486f60b544ad7b593af5
